examples/plot_simulate_alpha.py

```diff
@@ -29,32 +29,30 @@
 print(params)
 
 ###############################################################################
-# Now, we update a few parameters
+# Remove all evoked proximal feed parameters
+evoked_prox_keys = params['*evprox*'].keys()
+for key in evoked_prox_keys:
+    del params[key]
+
+# Remove all evoked distal feed parameters
+evoked_dist_keys = params['*evdist*'].keys()
+for key in evoked_dist_keys:
+    del params[key]
```

Comment:
```suggestion
for key in params['*evdist*']:
    params.pop(key)
```

Reply:
Any particular reason why pop is better than del in this situation? I thought pop was specifically designed to capture and return the key after removing it from the dict?

Reply:
umm ... not really. You can use `del`